### PR TITLE
Feature/villager trading

### PR DESCRIPTION
**Pull request by Huskykyle**
progress for #249 and complete #247

### CODE REVIEW

File: overrides/scripts/villager_trades_removed.zs [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+<profession:minecraft:toolsmith>,
+<profession:minecraft:weaponsmith>
+];
+
+for prof in villagerPro 
+{
+    villagerTrades.removeTradesSelling(prof, 1, <item:minecraft:emerald>);
+}
+
+val toBuy = [<item:farmersdelight:barbecue_stick>,
```
Comment (Jonah-Hansen) [inline overrides/scripts/villager_trades_removed.zs:85]: things inside brackets should start on a new line, indented to show it is nested inside the brackets

so 
```js
val toBuy = [
  <item>
]
```

File: overrides/scripts/villager_trades_removed.zs [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+       <item:farmersdelight:pasta_with_meatballs>,
+       <item:farmersdelight:pasta_with_mutton_chop>,
+       <item:farmersdelight:roast_chicken>,
+       <item:farmersdelight:roasted_mutton_chops>,
+       <item:farmersdelight:squid_ink_pasta>,
+       <item:farmersdelight:steak_and_potatoes>,
+       <item:farmersdelight:honey_glazed_ham>];
+
+for food in toBuy
+{
```
Comment (Jonah-Hansen) [inline overrides/scripts/villager_trades_removed.zs:124]: you may see differing opinions but I always prefer opening brackets on same line. 

```js
for food in toBuy {
  // your code
}
```

**Review by Jonah-Hansen — CHANGES_REQUESTED** [review]
just some formatting fixes

please add indentation anywhere you open a bracket.

for example line 80-83 is good, but you need indentation on the others.
2 or 4 spaces doesn't really matter just be consistent. or just the `TAB` key on the keyboard.

**Comment by Jonah-Hansen** [comment]
@GildedToxin so are we still wanting to add this?

**Comment by GildedToxin** [comment]
Idk, I know @RichPrune wants villager stuff at somepoint so maybe just keep it here till then for reference? or is that bad pratice?

**Comment by Jonah-Hansen** [comment]
imma just close it for now if we want the code we can still find it from this pr

**Pull request closed without merging**